src/products.js

```diff
@@ -139,6 +139,7 @@ function calculateVariation(input, options) {
       variation.sale_price = variant.sale_price || product.sale_price;
       variation.orig_price = variant.orig_price || product.orig_price;
       variation.stock_status = variant.stock_status || product.stock_status;
+      variation.images = variant.images || product.images;
```

Comment:
@by-tim you need to check if `variant.images` is not empty as well because an empty array would be truthy however we shouldn't use an empty array